src/Database/Driver/PDODriverTrait.php

```diff
@@ -94,9 +94,8 @@ public function isConnected()
                 $connected = false;
             }
         }
-        $this->connected = !empty($connected);
```

Comment:
Now I know what you mean. It was (or seemed like) a dead property.
Can you find any usage of it in any class that uses the trait? This was a leftover, so it looks like. Please confirm.